policyengine_us/reforms/states/ny/wftc/ny_working_families_tax_credit.py

```diff
@@ -434,6 +437,24 @@ def formula(tax_unit, period, parameters):
             household_credit = tax_unit("ny_household_credit", period)
             return max_(0, total_credit - household_credit)
 
+    class ny_exemptions_dependent(Variable):
+        value_type = bool
+        entity = Person
+        label = "Dependent under the New York exemptions definition"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://www.nysenate.gov/legislation/laws/TAX/616"
+        defined_for = StateCode.NY
+
+        def formula(person, period, parameters):
+            dependent = person("is_tax_unit_dependent", period)
```

Comment:
Parameters require `if` not `where`
```suggestion
            p = parameters(period).gov.contrib.states.ny.wftc.exemptions
            if p.in_effect:
                child_dependent = person("is_child_dependent", period)
                wftc_eligible_child = person("ny_wftc_eligible_child", period)
                return child_dependent & ~wftc_eligible_child
            return person("is_tax_unit_dependent", period)
```